content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,556 @@
+---
+slug: 32
+title: 32/RLN
+name: Rate Limit Nullifier
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors: 
```

Comment:
I'd like to suggest the following contributors
Barry (don't know his contact), Sanaz Taheri <sanaz@status.im>, and Oskar Thorén <oskar@status.im>

Reply:
Added, will ask Barry for contact info and will add that.